spec/support/spec/platform_data.rb

```diff
@@ -56,6 +56,12 @@ def platform_data(p, d)
       config_path: '/opt/app/patroni/etc/postgresql.yml',
     },
   }
+  sanitized_platform = if %r{(RedHat|CentOS|AlmaLinux|Rocky|OracleLinux)-(\d+)}.match(p)
+                         "RedHat-#{Regexp.last_match(2)}"
+                       else
+                         p
+                       end
+  puts sanitized_platform
   default = data['default'][d]
-  data.fetch(p, data['default']).fetch(d, default)
+  data.fetch(sanitized_platform, data['default']).fetch(d, default)
```

Comment:
I had to wrap my head around this, but isn't this equal and easier to follow?
```suggestion
  data.dig(sanitized_platform, d) || data['default'][d]
```

Reply:
mhm that could be the same. I didn't have the muse to dig too deep into it.

Reply:
lets test https://github.com/voxpupuli/puppet-patroni/pull/72